src/registries/side_panel_registry.ts

```diff
@@ -62,6 +63,11 @@ sidePanelRegistry.add("DataValidation", {
   Body: DataValidationPanel,
 });
 
+sidePanelRegistry.add("DataValidationEditor", {
+  title: _t("Data validation editor"),
```

Comment:
I would have the same title. Such that the title is the same no matter how this side panel is open
```suggestion
  title: _t("Data validation"),
```